articles/active-directory/external-identities/add-users-administrator.md

```diff
@@ -112,14 +112,16 @@ If a guest user has not yet redeemed their invitation, you can resend the invita
 3. Under **Manage**, select **Users**.
 5. Select the user account.
 6. Under **Manage**, select **Profile**.
-7. If the user has not yet accepted the invitation, a **Resend invitation** option is available. Select this button to resend.
+7. If the user has not yet accepted the invitation, a **Resend invitation** option is available. Select **manage** to open the **Manage invitations** menu.
+
+   ![B2B_ResendNew](https://user-images.githubusercontent.com/80779693/111352250-e241e200-867b-11eb-9adb-3b484cd5992a.png)
```

Comment:
Thanks for the screenshot for reference - you can remove it for now and I'll follow up with a formatted one (we have specific guidelines for size, etc.